src/App.vue

```diff
@@ -20,7 +20,19 @@
         {{ $t('general.forgotPassword.title') }}
       </router-link>
 
+      <nav-dropdown username="someusername" avatar=""/>
```

Comment:
Here I thought that it would not be necessary to pass the whole user as a prop, the username and the avatar should be sufficient